ts/features/idpay/code/onboarding/screens/IDPayCodeDisplayScreen.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Banner,
+  Body,
+  ButtonSolid,
+  IOColors,
+  IOSpacingScale,
+  Label,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { StyleSheet, View } from "react-native";
+
+import { useNavigation } from "@react-navigation/native";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { NewH3 } from "../../../../../components/core/typography/NewH3";
+import TopScreenComponent from "../../../../../components/screens/TopScreenComponent";
+import I18n from "../../../../../i18n";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+
+const CodeDigitDisplayBox = ({ digit }: { digit: string }) => (
+  <View style={styles.codeDigit}>
+    <NewH3>{digit}</NewH3>
+  </View>
+);
+const CodeDisplayComponent = ({ code }: { code: string }) => (
+  <View style={styles.codeDisplay}>
+    {code.split("").map((digit, index) => (
+      <CodeDigitDisplayBox key={index} digit={digit} />
+    ))}
+  </View>
+);
+
+export const IDPayCodeDisplayScreen = () => {
```

Comment:
Same for file name: `IdPayCodeDisplayScreen.tsx`
```suggestion
export const IdPayCodeDisplayScreen = () => {
```

Reply:
I'm pretty sure we had decided a while ago to use `IDPay` notation to reference the feature, also that's the name that is being used in the redux-saga PR

Reply:
We are using `IdPay` prefix in the `code` feature as the new standard.

Reply:
not a big fan, but addressed nonetheless 👍